README.rst

```diff
@@ -194,123 +147,66 @@ the computational workflow steps and the expected outputs:
     inputs:
       files:
         - code/worldpopulation.ipynb
-        - inputs/World_historical_and_predicted_populations_in_percentage.csv
+        - data/World_historical_and_predicted_populations_in_percentage.csv
       parameters:
         notebook: code/worldpopulation.ipynb
-        input_file: inputs/World_historical_and_predicted_populations_in_percentage.csv
-        output_file: outputs/plot.png
+        input_file: data/World_historical_and_predicted_populations_in_percentage.csv
+        output_file: results/plot.png
         region: Africa
         year_min: 1500
         year_max: 2012
     outputs:
       files:
-       - outputs/plot.png
+       - results/plot.png
     workflow:
       type: serial
       specification:
         steps:
           - environment: 'reanahub/reana-env-jupyter'
             commands:
-              - mkdir -p outputs && papermill $notebook /dev/null -p input_file ${input_file} -p output_file ${output_file} -p region $region -p year_min ${year_min} -p year_max ${year_max}
+              - mkdir -p results && papermill $notebook /dev/null -p input_file ${input_file} -p output_file ${output_file} -p region $region -p year_min ${year_min} -p year_max ${year_max}
 
 In case you are using CWL or Yadage workflow specifications:
 
 - `reana.yaml using CWL <reana-cwl.yaml>`_
 - `reana.yaml using Yadage <reana-yadage.yaml>`_
 
-We proceed by installing the REANA command-line client:
+We can now install the REANA command-line client, connect to a REANA cloud
+instance where we want to run the analysis, upload the analysis input data and
+code, start the analysis and, after it finishes, download the resulting plots:
```

Comment:
Only part I would rephrase for brevity, to something like: "We can now install the REANA command-line client, run the analysis and download the resulting plots:"
Not mentioning every step, which could be part of the more extensive REANA Client docs.